installer/build-script.sh

```diff
@@ -83,7 +83,7 @@ if [ ! command -v dkms &> /dev/null ]; then
         /usr/bin/dnf install -y dkms.noarch
     elif [ -f /usr/bin/apt ]; then
         /usr/bin/apt-get -q -y install -y dkms.noarch
-    else [ -f /usr/bin/yum ]; then
+    else [ -f /usr/bin/yum ]
```

Comment:
Do you really want :

else test -f /usr/bin/yum

?

Maybe as srini suggests:

elif [ -f /usr/bin/yum ]; then
   ...
   ...
fi

Reply:
Right, it does look suspicious